### ISSUE TITLE
OTTServoTester with LCD Keypad Shield

### ISSUE THREAD

**Issue by freetoair** [MajicDesigns/OTTServoTester#1]
Hi Marco, I tried to modify your excellent project to work with "LCD Keypad Shield" but without success.
The part with the parallel LCD display works correctly, but the control of the analog keys is a big problem for me.
If there is some way to do it without a lot of involvement, I think it would mean a lot to many who would be happy to use this tester. If it means anything, I can upload my modified Sketch.

regards,  freetoair.

**Comment by MajicDesigns** [comment]
Use the analog keys tester example in the UISwitch library to work out the analog values for the switches on your keypad shield. The substitute these values for those in the OTTtester code.

**Comment by freetoair** [comment]
Yes, that's exactly what I tried to do, but as you can see from my program in debug mode when selecting in the "setup" menu, it's as if some key was pressed, but it wasn't!
Here I am sending the variant that I tried.
[OTTServoTester_LCDkeySheld.zip](https://github.com/MajicDesigns/OTTServoTester/files/11778641/OTTServoTester_LCDkeySheld.zip)

**Comment by MajicDesigns** [comment]
"as you can see from my program in debug mode" - not really, I can't see anything that is debug output.

One area that often causes you problem is if you are trying to power the servos from the Arduino board. As soon as the servos start to move they will cause a dip on the voltage on the board and noise in the electonic components power supply which makes them behave erratically.
